app/views/shared/_photographer.html.erb

```diff
@@ -1,85 +1,85 @@
-<h2>Hello, <%= current_user.first_name %></h2>
+<h2 style="background-color: #0fabbc; font-size: 30px; color: white; padding: 10px; max-width: fit-content;; border-radius: border-radius: 15px 50px 30px; border-radius: 1px 29px 28px 1px;">Hello, <%= current_user.first_name %></h2>
   <div class="container">
     <div class="vertical-tabs">
       <ul class="nav nav-tabs" role="tablist">
         <li class="nav-item">
-          <a class="nav-link active" data-toggle="tab" href="#pag1" role="tab" aria-controls="My Clients">My Clients</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" data-toggle="tab" href="#pag2" role="tab" aria-controls="Event Manager">Event Manager</a>
+          <a class="nav-link active" data-toggle="tab" href="#pag2" role="tab" aria-controls="Event Manager">Event Manager</a>
         </li>
         <li class="nav-item">
+          <a class="nav-link " data-toggle="tab" href="#pag1" role="tab" aria-controls="My Clients">My Clients</a>
+        <li class="nav-item" >
           <a class="nav-link" data-toggle="tab" href="#pag3" role="tab" aria-controls="Portfolio">Portfolio</a>
         </li>
-        <li class="nav-item">
-          <a class="nav-link" data-toggle="tab" href="#pag4" role="tab" aria-controls="Messages">Messages</a>
-        </li>
+
       </ul>
 
       <div class="tab-content">
 
-        <div class="tab-pane active" id="pag1" role="tabpanel">
+        <div class="tab-pane " id="pag1" role="tabpanel">
           <div class="sv-tab-panel">
-            <h3>My Clients</h3>
+      <div class="event-show-card mt-4" style="background-color: white;">
                 <% @bookings.each do |booking| %>
                   <%= render 'shared/client_booking', booking: booking %>
                 <% end %>
           </div>
         </div>
-
-        <div class="tab-pane" id="pag2" role="tabpanel">
+      </div>
+        <div class="tab-pane active" id="pag2" role="tabpanel">
           <div class="sv-tab-panel">
-            <div class="d-flex" style="justify-content: space-between;">
-              <div class="my-event">
-              <h3>My Events</h3>
-              </div>
-              <div id="sticky">
-              <h2>Booked Events</h2>
-              <li>evernt 1</li>
-              <li>evernt 1</li>
-              <li>evernt 1</li>
-              <li>evernt 1</li>
+            <!-- Nested Tabs -->
+            <div class="container">
+                <ul class="nav nav-tabs" role="tablist">
+
+                  <li class="nav-item">
+                    <a class="nav-link active" data-toggle="tab" href="#pag2-1" role="tab" aria-controls="My Clients">My Events</a>
```

Comment:
@Jadiman17 is the aria-controls supposed to be "My Clients" or refer to events in this line?